.github/workflows/main.yml

```diff
@@ -15,18 +15,33 @@ env:
   IMAGE_NAME: php74-fpm
 
 jobs:
-  lint-docker-file:
+  codespell:
+    name: codespell
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: lint dockerfile
-      uses: brpaz/hadolint-action@master
-      with:
-        dockerfile: Dockerfile
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Codespell
+        uses: codespell-project/actions-codespell@master
```

Comment:
für was brauchen wir das genau?

Reply:
Right away from https://github.com/codespell-project/codespell:

> Fix common misspellings in text files. It's designed primarily for checking misspelled words in source code, but it can be used with other files as well. It does not check for word membership in a complete dictionary, but instead looks for a set of common misspellings. Therefore it should catch errors like "adn", but it will not catch "adnasdfasdf". This also means it shouldn't generate false-positives when you use a niche term it doesn't know about.

Reply:
example: https://github.com/tmtde/php81-fpm/commit/34aecaca0b23a9b6aacab5b513f26cae3cd6dbd3#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R11